src/core_ocean/shared/mpas_ocn_adcReconstruct.F

```diff
@@ -183,9 +183,10 @@ subroutine compute_ADC_tends(nCells,nVertLevels, nTracers, dt,activeTracers, uve
       real(kind=RKIND),dimension(nVertLevels,nCells) :: areaFractionMid, tumdMid, McMid, wumdMid, sumdMid
 
       real(kind=RKIND) :: SwU, SwD, tau_sfc, d_sqrt_wp2_dz, tp2, sp2, min_wp2_sfc_val, wp2_splat_sfc_correction
-      real(kind=RKIND) :: min_wps_sfc_val
+      real(kind=RKIND) :: min_wps_sfc_val, adcRound
 
       min_wps_sfc_val = 1.0E-10_RKIND
+      adcRound = 10.0_RKIND**config_adc_decimals_to_keep
```

Comment:
@katsmith133 I would suggest moving this to `mpas_ocn_vmix_adc.F` in the init section there is no need to do this every time we call the routine.  You'll have to add adcRound to mpas_ocn_turbulence.F too

Reply:
@vanroekel Could it not all go in to `mpas_ocn_turbulence.F`? Seems like it fits well in being set somewhere in here, rather than in `mpas_ocn_vmix_adc.F`:
https://github.com/vanroekel/MPAS-Model/blob/3e01de373ab6ae5e57343436fd7a24bcb642396b/src/core_ocean/shared/mpas_ocn_turbulence.F#L149-L176

Reply:
Right, that is a much better idea.  Do put it all there instead of splitting.

Reply:
Done!